t/core_bools.t

```diff
@@ -0,0 +1,68 @@
+use strict;
+use warnings;
+use JSON::PP;
+use Test::More;
+BEGIN {
+  # this is only for JSON.pm
+  plan skip_all => 'no support for core boolean options'
+    unless JSON::PP->can('CORE_BOOL');
+}
+
+plan tests => 18;
+
+my $json = JSON::PP->new;
+
+is $json->get_core_bools, !!0, 'core_bools initially false';
+
+my $ret = $json->core_bools;
+is $ret => $json, "returns the same object";
+
+SKIP: {
+    skip "core_bools option doesn't register as true without core boolean support", 1
+        unless JSON::PP::CORE_BOOL;
```

Comment:
The docs, PR description and code say it does.

Reply:
Fixed.